docs/source/concepts/ledger-model/ledger-structure.rst

```diff
@@ -257,7 +267,19 @@ following ledgers are not.
    Painter stealing Alice's IOU. Note that the ledger would be
    intuitively permissible if it was Alice performing the last commit.
 
+.. figure:: ./images/failed-key-assertion.svg
```

Comment:
I would remove the maintainer annotations from these examples.

Reply:
So you'd write this as `PaintOffer A &P Bank &P123`? But then we have to change the `&` notation in the privacy section later.

Reply:
We keep the & and @ as discussed.